helpdesk/timesheet_and_invoice/invoice_time.rst

```diff
@@ -1,126 +1,111 @@
 
-Step 2 : gather a helpdesk team
--------------------------------
+| Go to :menuselection:`Sales --> Configuration --> Settings` and enable *Units of Measure* to
+  optionally be able to choose *hours* (for example) as the unit of measure of your service.
+| Then, go to :menuselection:`Sales --> Products --> Products`, create or edit an existing one, and
+  set its *Product Type* as *Service*.
 
-To set a team in charge of the helpdesk, go to :menuselection:`Helpdesk
---> Configuration --> Helpdesk Teams` and create a new team or select
-an existing one. On the form, tick the box in front of *Timesheet on
-Ticket* to activate the feature. Make sure to select the helpdesk
-project you have previously created as well.
-
-.. image:: media/invoice_time02.png
+.. image:: media/product_type_unit.png
    :align: center
+   :alt: View of a product's form emphasizing the product type and unit of measure fields in Odoo
+         Sales
 
-Step 3 : launch the helpdesk service
-------------------------------------
-
-Finally, to launch the new helpdesk service, first go to
-:menuselection:`Sales --> Configuration --> Settings` and make sure
-that the *Units of Measure* feature is activated.
+Now, select the invoicing management you would like to have under the *Sales* tab. We recommend the
+following configuration:
 
-.. image:: media/invoice_time03.png
+.. image:: media/product_invoicing.png
    :align: center
+   :alt: View of a product form and the invoicing options under the tab sales in Odoo Sales
 
-Then, go to :menuselection:`Products --> Products` and create a new
-one. Make sure that the product is set as a service.
+This configuration ensures that the customer is invoiced by the number of hours predicted in the
+sales order, meaning that less or extra hours recorded are not taken into account. It also ensures
+that every time a sales order is confirmed, a new task is created under the right project,
+automating the process.
 
-.. image:: media/invoice_time04.png
-   :align: center
-
-.. note::
-   Here, we suggest that you set the *Unit of Measure* as
-   *Hour(s)*, but any unit will do.
+.. tip::
+   We recommend setting up a specific project, as it was done for this flow example. The important
+   thing to remember is that the sales order item needs to be set on the corresponding project or
+   task, in order to reinvoice the time spent on a ticket.
 
-Finally, select the invoicing management you would like to have under
-the *Sales* tab of the product form. Here, we recommend the following
-configuration :
+Prevision an invoice and record time
+====================================
 
-.. image:: media/invoice_time05.png
-   :align: center
+Step 1: Place an order
+----------------------
 
-Now, you are ready to start receiving tickets !
+Go to :menuselection:`Sales --> Orders --> Orders` and create one for the helpdesk service product
+you have previously set up, with the customer who needs the ticket to be open. Set the number of
```

Comment:
```suggestion
you have previously set up, with the customer who needs the ticket to be opened. Set the number of
```
Not sure but I do prefer suggesting even if I'm wrong ^^

Reply:
it is a tricky one hehe but I think you are right. Merciii!